### ISSUE TITLE
spam detection code using tensor code

### ISSUE THREAD

**Issue by PavaniPudi23** [PavaniPudi23/NEURALNetworks#2]
import tensorflow as tf
import numpy as np

def relu(x):
    return tf.nn.relu(x)

def sigmoid(x):
    return tf.nn.sigmoid(x)

class FeedForward2HL_TF(tf.keras.Model):
    def __init__(self):
        super().__init__()
        self.W1 = tf.Variable([0.5, -0.2, 0.3], dtype=tf.float32)
        self.W2 = tf.Variable([0.4, 0.1, -0.5], dtype=tf.float32)
        self.Wout = tf.Variable(1.0, dtype=tf.float32)

    def call(self, x):
        z1 = tf.reduce_sum(x * self.W1)
        h1 = relu(z1)
        z2 = h1 * tf.reduce_sum(self.W2)
        h2 = sigmoid(z2)
        z3 = h2 * self.Wout
        output = sigmoid(z3)
        return output * 100

x = tf.constant([1.0, 0.0, 1.0])
model = FeedForward2HL_TF()
prob = model(x).numpy()
print(f"The mail is {prob:.2f}% likely to be spam.")
